app/controllers/fast_controller.rb

```diff
@@ -20,55 +20,40 @@ def show
   private
 
   def lookups(query)
-    # Try first without wildcard.
     result = lookup(query)
     return result if result.failure?
 
-    suggestions = result.value!
-    if suggestions.size < num_records
-      # If not enough results then add wildcard.
-      wildcard_result = lookup(query, wildcard: true)
-      wildcard_suggestions = wildcard_result.value_or([])
-    end
-
-    Success(merge_suggestions(suggestions, wildcard_suggestions, query))
+    Success(result.value!)
   end
 
-  # rubocop:disable Metrics/AbcSize
-  def lookup(query, wildcard: false)
+  def lookup(query) # rubocop:disable Metrics/AbcSize
     resp = lookup_connection.get do |req|
-      req.params['q'] = wildcard ? "keywords:(#{query}*)" : "keywords:(#{query})"
+      req.params['query'] = query
+      req.params['queryIndex'] = 'suggestall'
+      req.params['queryReturn'] = 'idroot,suggestall,tag'
+      req.params['suggest'] = 'autoSubject'
```

Comment:
@arcadiafalcone @amyehodge @edsu These are my best guesses based on https://www.oclc.org/developer/api/oclc-apis/fast-api/assign-fast.en.html and #3581

Reply:
Confirmed by looking at our initial implementation in 2021.